loader.js

```diff
@@ -80,8 +95,10 @@ async function processLayout(options, frontMatter, content, resourcePath, scans)
 
   // If no layout is provided and the default layout setting is not on, return the
   // content directly.
-  if (!mergedFrontMatter.layout && !pluginOpts.defaultLayout)
+  if (!mergedFrontMatter.layout && !pluginOpts.defaultLayout) {
     return content
+  }
+    
```

Comment:
What's going on with this change?

Reply:
I believe it's curly braces around the if() added by Prettier, but it only shows the closing bracket?